dazel/lib/src/bazelify/build.dart

```diff
@@ -154,6 +161,15 @@ class BuildFile {
       buffer.writeln('load("$_vmBzl", "dart_vm_binary")');
       buffer.writeln();
     }
+    final buildersUsed =
+        libraries.expand((l) => l.builders?.keys ?? const <String>[]);
+    if (buildersUsed.isNotEmpty) {
+      buffer
+        ..writeln('# Dazel: Using codegen.')
+        ..writeln(
+            'load(":codegen.bzl", ${buildersUsed.map((b) => '"$b"').join(', ')})')
```

Comment:
I think we need to make sure we specify the repo here (should be the root one I think?). If some package I depend on is using builders, the way its set up now the rules are defined in the parent repo (as far as I can tell). Therefore I think this won't work?

I could be reading things wrong though so lmk.